docs/atomic-upgrades.md

```diff
@@ -30,14 +30,14 @@ the remote server.  Suppose we're tracking a ref named
 which contains a SHA256 checksum.  This determines the tree to deploy,
 and `/etc` will be merged from currently booted tree.
 
-If we do not have this commit, then, then we perform a pull process.
+If we do not have this commit, then we perform a pull process.
 At present (without static deltas), this involves quite simply just
 fetching each individual object that we do not have, asynchronously.
 Put in other words, we only download changed files (zlib-compressed).
 Each object has its checksum validated and is stored in `/ostree/repo/objects/`.
 
-Once the pull is complete, we have all the objects locally
-we need to perform a deployment.
+Once the pull is complete, we have, locally, all the objects that we need
```

Comment:
Is there a reference for this type of change?  Pulling up
https://owl.purdue.edu/owl/general_writing/punctuation/commas/extended_rules_for_commas.html
for example, I'd say "locally" here is an essential part of the sentence and so shouldn't be "comma quoted", right?

Perhaps just say:
"Once the pull is complete, we have downloaded all of the objects that we need to perform a deployment."

Reply:
Yeah, was thinking to myself about where to put "locally", since the position of it in the original was quite awkward to me; but if it's not too important, I guess we can just remove it :)

Regarding the use of commas, I followed Rule 3 in your link. Though I might be misunderstanding that rule.